docs/rules/no-await-in-loop.md

```diff
@@ -0,0 +1,89 @@
+# Disallow accidental serial execution of parallelizable operations (no-await-in-loop)
```

Comment:
I'd rather update this to say "Disallow await in loops" to match `meta.description`. The simpler explanation is much clearer.